src/style.css

```diff
@@ -121,3 +121,13 @@ body {
     width: 100%;
   }
 }
+
+@media screen and (max-width: 480px) {
+  .todo-list {
+    width: 100%;
+  }
+
+  .tasks {
+    grid-template-columns: repeat(auto-fill, minmax(320px, 1fr));
+  }
+}
```

Comment:
- [ ] I would recommend fixing a UI issue because it cause a problem and I am unable to see `task 1` and `task 2`.

![Screenshot from 2021-12-03 20-55-25](https://user-images.githubusercontent.com/22434660/144631477-402fb4e2-4c5e-43ef-bde2-ac999a790c33.png)